README.md

```diff
@@ -54,8 +54,43 @@ If all you want to do is use an already trained model for doing speech-to-text,
 
```

Comment:
There's also one pip install before this one, right after the gif. I'd guess you should also add a comment about virtualenv's there too, maybe just suggesting virtualenv use and referring to the instructions below.

I just worry as the first pip install is before the virtual env stuff, people will, reading from top to bottom, do pip installs without a virtual env.